HousingSearchListener.Tests/V1/Gateway/EsGatewayTests.cs

```diff
@@ -149,5 +164,241 @@ public async Task IndexTenureTestCallsEsClient()
             _cleanup.Add(async () => await _testFixture.ElasticSearchClient.DeleteAsync(new DeleteRequest("tenures", tenure.Id))
                                                                            .ConfigureAwait(false));
         }
+
+        [Fact]
+        public async Task UpdatePersonBalanceTestCallsEsClient()
+        {
+            var sut = new EsGateway(_testFixture.ElasticSearchClient);
+            var person = CreatePerson();
+            var account = CreateAccount();
+
+            var createdResponse = await sut.IndexPerson(person).ConfigureAwait(false);
```

Comment:
There are a few shoulds and calls here mixed with one another - and the name of the test is called 'CallsEsClient' - is this test perhaps worth breaking down in multiple tests to make it a bit cleaner?

Reply:
@EdwardAnoghena Maybe make sense to change tests style to BDD?